app/controllers/admin/site_steps_controller.rb

```diff
@@ -157,6 +157,11 @@ def update
         else
           @site.form_step = 'template'
 
+          if @site.site_template.nil?
```

Comment:
I don't think so, because the model validations don't take into account the step-by-step behavior we have. It would prevent saving the "site" model on steps that don't even allow the user to select the template so we either:
- make the model validator aware of the site step
- hack it this way

Since I don't know how to do the first one (and it does seem a bit overkill), hacky way it is! Unless you have an alternative, ofc. let me know